src/AppController.m

```diff
@@ -2496,9 +2496,6 @@ -(void)viewArticlePages:(id)sender inPreferredBrowser:(BOOL)usePreferredBrowser
             }
 		}
 		[self openURLs:urls inPreferredBrowser:usePreferredBrowser];
-		
-		if (!db.readOnly)
-            [articleController markReadByArray:articlesWithLinks readFlag:YES];
 	}
```

Comment:
Welcome back in the Vienna team, Jeff!
You probably rebased from your personal branch.